src/scss/app.js

```diff
@@ -0,0 +1,9 @@
+import createBtn from './btn';
+
+function app() {
+  const root = document.querySelector('#app');
+  const btn = createBtn('play the game');
+  root.appendChild(btn);
+}
+
+export default app;
```

Comment:
Why did you put a js component to `scss` styles directory? Put it to `src/components/app.js`. We should keep js and styles separate.

Reply:
I understand